packages/cosmic-swingset/src/kernel-stats.js

```diff
@@ -8,6 +8,8 @@ import {
   KERNEL_STATS_UPDOWN_METRICS,
 } from '@agoric/swingset-vat/src/kernel/metrics.js';
 
+import '@agoric/swingset-vat';
```

Comment:
Interesting. I think swingset follows the `exported.js` pattern. It might be a side effect of the recent hybrid ambient / exported types approach that this actually even works.

```suggestion
import '@agoric/swingset-vat/exported.js';
```

Reply:
> Interesting. I think swingset follows the `exported.js` pattern. It might be a side effect of the recent hybrid ambient / exported types approach that this actually even works.

Argh.  There's no satisfactory way to get the type of the `RunPolicy` unless I shuffle SwingSet types around.  I think I'll leave it as `any`.

`exported.js` currently only pulls in the timer vat's types, and even just that results in:

```
snow:cosmic-swingset michael$ yarn lint
yarn run v1.22.17
$ run-s --continue-on-error lint:*
$ tsc --maxNodeModuleJsDepth 3 -p jsconfig.json
../SwingSet/src/vats/timer/types.js:13:23 - error TS2304: Cannot find name 'ERef'.

13  *             waker: ERef<TimerWaker>
                         ~~~~

../SwingSet/src/vats/timer/types.js:17:23 - error TS2304: Cannot find name 'ERef'.

17  * @property {(waker: ERef<TimerWaker>) => Array<Timestamp>} removeWakeup
                         ~~~~

../SwingSet/src/vats/timer/types.js:35:9 - error TS2304: Cannot find name 'Notifier'.

35  * ) => Notifier<Timestamp>} makeNotifier
           ~~~~~~~~

../SwingSet/src/vats/timer/types.js:66:23 - error TS2304: Cannot find name 'ERef'.

66  * @property {(waker: ERef<TimerWaker>) => Timestamp} schedule
                         ~~~~


Found 4 errors.
```

Reply:
Ugh ambient types need to go!